tests/info_extraction/InfoScopeTest.py

```diff
@@ -0,0 +1,129 @@
+import os
+import unittest
+
+from coala_quickstart.info_extraction.Info import Info
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+from coala_quickstart.info_extraction.InfoScope import InfoScope
+
+
+class InfoScopeTest(unittest.TestCase):
+
+    def setUp(self):
+
+        class DummyInfoExtractor(InfoExtractor):
+
+            def parse_file(self, file_content):
+                return file_content
+
+            def find_information(self, fname, parsed_file):
+                return [Info(
+                    fname,
+                    'Dummy information it is!')]
+
+        class NoInfoExtractor(InfoExtractor):
+
+            def parse_file(self, fname, file_content):
+                return file_content
+
+            def find_information(self, fname, parsed_file):
+                return []
+
```

Comment:
> No need to do it separately for every tests.

I didn't get you :sweat_smile: I've defined them inside `setUp` just to avoid defining them again and again for every test. And I've seen similar definitions at other locations in coala repos as well :)